### DISCUSSION TITLE
Light Mode

### DISCUSSION THREAD

**Discussion by Ansh4011** [aleksey-hoffman/sigma-file-manager#117] [Ideas]
Just making a request, can we get the light mode in the next update or anytime sooner? Light mode has a different and wonderful aura and of course the positive vibes, plus point, they are good for the eyes too! Thanks :)

**Comment by aleksey-hoffman** [comment]
Isn't this a duplicate of #74?

**Reply by Ansh4011** [reply]
The other discussion was showcasing the idea... 
Here I was requesting for the same, cause a Light Mode is a must for many users including me! This becomes a great drawback! I also surveyed this in my knowns.

**Comment by Ansh4011** [comment]
Uhh.... I hope you know about all this stuff going on where sigma file manager is available to some other site (even the download link is not of github).
 
https://www.chip.de/downloads/Sigma-File-Manager_183635503.html
![image](https://user-images.githubusercontent.com/84930340/126889047-c0b402d4-1465-48f4-95a3-bd2f3082b89c.png)

**Reply by aleksey-hoffman** [reply]
@Ansh4011 thank you for the information. 
I checked the file on their website, they are using the same file, which means they didn't modify it in any way and it's safe to install.

**Reply by Ansh4011** [reply]
Oh okay! Thanks buddy :)
